### ISSUE TITLE
[Whitebark's spirit] [Eversong Woods] [Entangle spell]

### ISSUE THREAD

**Issue by Gedlund** [Atlantiss/NetherwingBugtracker#241]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 
Whitebark's spirit appears correctly when attempting to complete a quest but when he casts entangling roots they are not applied to your character.

**Current behaviour**:
Whitebark's spirit casts entangling roots but dosen't root players.

**Expected behaviour**: 
Whitebark's spirit entangle is cast and roots a player

**Server Revision**:

**Comment by Evolvee** [comment]
are you sure u didnt resist it? it works fine for me

**Comment by Gedlund** [comment]
Yeah im quite sure it wasnt a resist, was on a belf female pally if it makes any difference. he casted it twice and nothing happened
